tests/plots/test_mapping.py

```diff
@@ -10,6 +10,20 @@
 from metpy.plots.mapping import CFProjection  # noqa: E402
 
 
+def test_inverse_flattening_0():
+    """Test new code for dealing the case where inverse_flattening = 0."""
+    attrs = {'grid_mapping_name': 'lambert_conformal_conic', 'earth_radius': 6367000,
```

Comment:
The original issue in #1844 was for a case where `'earth_radius'` was NOT present, so we should probably replace that with `'semimajor_axis'` and `'semiminor_axis'` as was done there. As of right now, your test isn't actually triggering the new code.